msteams-platform/toolkit/debug-background-process.md

```diff
@@ -10,170 +10,57 @@ ms.date: 03/03/2022
-|Azure functions |.NET Core SDK version|• If .NET Core SDK isn't installed or the version  doesn't match the requirement, the Toolkit installs .NET Core SDK for Windows and macOS in `~/.fx/bin/dotnet`.</br> • For Linux, the local debug terminates.|
-|Azure functions | Azure functions binding extensions defined in `api/extensions.csproj`| If Azure functions binding extensions isn't installed, the Toolkit installs Azure functions binding extensions.|
-|NPM packages| NPM packages for tab app, bot app, message extension app, and Azure functions|If NPM isn't installed, the Toolkit installs all NPM packages.|
-|Bot and message extension | Ngrok |Toolkit starts Ngrok to create a HTTP tunnel for bot and message extension. |
-
-> [!NOTE]
-> If tab, bot, message extension, and Azure functions ports are unavailable, the local debug terminates.
-
-Use the following .NET Core versions:
-
-| Platform  | Software|
-| --- | --- |
-|Windows, macOs (x64), Linux | **3.1 (recommended)**, 5.0, 6.0 |
-|macOs (arm64) |6.0 |
-
-> [!NOTE]
-> If the development certificate for localhost isn't installed for tab in Windows or MacOS, the Teams Toolkit prompts you to install it.</br> -->
+* Teams Toolkit checks if custom app uploading or sideloading for your developer tenant is turned on. If it isn't turned on, the debugging terminates.
+* Teams Toolkit checks if ports are available. If tab, bot, message extension, and Azure Functions ports are unavailable, the debugging terminates.
 
 When you select **Start Debugging (F5)**, Teams Toolkit output channel displays the progress and result after checking the prerequisites.
 
-   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/prerequisites-debugcheck1.png" alt-text="Prerequisites check summary" lightbox="../assets/images/teams-toolkit-v2/debug/prerequisites-debugcheck1.png":::
-
-## Register and configure Teams app
-
-In the set-up process, Teams Toolkit prepares the following registrations and configurations for your Teams app:
-
-1. [Registers and configures Microsoft Azure Active Directory(Azure AD) app](#registers-and-configures-microsoft-azure-active-directoryazure-ad-app)
-
-1. [Registers and configures bot](#registers-and-configures-bot)
-
-1. [Registers and configures Teams app](#registers-and-configures-teams-app)
-
-### Registers and configures Microsoft Azure Active Directory(Azure AD) app
-
-1. Registers an Azure AD app.
-
-2. Creates a Client Secret.
-
-3. Exposes an API.
+:::image type="content" source="../assets/images/teams-toolkit-v2/debug/prerequisites-debugcheck.png" alt-text="Prerequisites check summary":::
```

Comment:
Update this image to v5 version, something like
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/37978464/5269b48a-6dce-439e-b773-a30320b1af69)